README.md

```diff
@@ -3,4 +3,4 @@ Instrumentation - A stats collection framework
 [![Build Status](https://travis-ci.org/census-instrumentation/instrumentation-java.svg?branch=master)](https://travis-ci.org/census-instrumentation/instrumentation-java) [![Build status](https://ci.appveyor.com/api/projects/status/v5dbthkucuewsu33/branch/master?svg=true)](https://ci.appveyor.com/project/instrumentationjavateam/instrumentation-java/branch/master)
 
 Instrumentation provides a framework to define and collect stats against metrics and to
-break those stats down across user-defined dimensions.
+break those stats down across user-defined dimensions.  The API is currently unstable.
```

Comment:
I would say something like (or something similar):

The library is in alpha stage and API is subject to change.